mlrun/kfpops.py

```diff
@@ -377,6 +393,8 @@ def mlrun_pipeline(
         cmd += ["-x", f"{xpram}={val}"]
     for input_param, val in inputs.items():
         cmd += ["-i", f"{input_param}={val}"]
+    for logging_configuration in returns:
+        cmd += ["--returns", f"{logging_configuration}"]
```

Comment:
Looking at params:
```
    for param, val in params.items():
        cmd += ["-p", f"{param}={val}"]
```
It seems to do the same thing, I simply followed, aren't we support white spaces in values of parameters?

Reply:
It's complicated, since this is going through several layers of translation, and is going into object specs etc. I don't think we expect params to have spaces in them, but since it's an array then I guess when serializing it may do the right thing and use `""` around each value in the list. So, let's assume it works (though I would be happier if you said "sure it works, I tested it!" 😏 )

Reply:
I works, I tested it!